scilpy/io/image.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from scilpy.utils.nibabel_tools import get_data
+
+
+def get_reference_info(reference):
+    """Get basic informations from a reference NIFTI file.
+    :param reference: Nibabel.nifti or filepath (nii or nii.gz)
+    """
+
+    # TODO remove/replace after stateful_tractogram PR is merged
+    _, nib_file = get_data(reference, return_object=True)
```

Comment:
In this specific case, using `get_data()` slows down computations. The data is loaded for absolutely nothing. There should be a lazy option or something similar.